scan/src/pages/ProposalHomePage.re

```diff
@@ -0,0 +1,224 @@
+module Styles = {
+  open Css;
+  let infoContainer =
+    style([
+      backgroundColor(Colors.white),
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(2), ~blur=`px(4), Css.rgba(0, 0, 0, 0.08))),
+      padding(`px(24)),
+      position(`relative),
+      Media.mobile([padding(`px(16))]),
+    ]);
+  let idContainer = {
+    style([
+      selector(
+        "> h4",
+        [marginLeft(`px(6)), Media.mobile([marginLeft(`zero), marginTop(`px(16))])],
+      ),
+    ]);
+  };
+  let badgeContainer = {
+    style([Media.mobile([position(`absolute), right(`px(16)), top(`px(16))])]);
+  };
+  let badge = color => {
+    style([backgroundColor(color), padding2(~v=`px(3), ~h=`px(10)), borderRadius(`px(50))]);
+  };
+};
+
+let getBadgeText =
+  fun
+  | ProposalSub.Deposit => "Deposit Period"
+  | Voting => "Voting Period"
+  | Passed => "Passed"
+  | Rejected => "Rejected"
+  | Failed => "Failed";
+
+let getBadgeColor =
+  fun
+  | ProposalSub.Deposit
+  | Voting => Colors.bandBlue
+  | Passed => Colors.green4
+  | Rejected
+  | Failed => Colors.red4;
```

Comment:
Let's make the Proposal Badge Component because the proposal index page also uses it.